src/webui/webapplication.cpp

```diff
@@ -273,7 +273,7 @@ void WebApplication::doProcessRequest()
             break;
         case QMetaType::QString:
         default:
-            print(result.toString(), Http::CONTENT_TYPE_TXT);
+            print(result.toString(), (Http::CONTENT_TYPE_TXT + QLatin1String("; charset=UTF-8")));
```

Comment:
Shouldn't you modify `Http::CONTENT_TYPE_TXT` instead? Seems qBittorrent Web API isn't supposed to return text in any other charset.

Reply:
>Shouldn't you modify Http::CONTENT_TYPE_TXT instead? 

IMO this way is a little inflexible but I think we can try it. It is done.